lib/rails_admin/config/fields/association.rb

```diff
@@ -57,6 +57,11 @@ def association # rubocop:disable TrivialAccessors
           @associated_collection_cache_all ||= (associated_model_config.abstract_model.count < 100)
         end
 
+        # Reader wheter association's elements can be removed
```

Comment:
J'aurais rajouté "or not" à la fin.
Mais je ne suis pas certain que ta phrase soit fausse :)